zulip_bots/zulip_bots/bots/bugzilla/doc.md

```diff
@@ -0,0 +1,30 @@
+# Bugzilla bot
+
+This bot allows to update directly Bugzilla from Zulip
+
+## Setup
+
+To use Bugzilla Bot, first set up `bugzilla.conf`. `bugzilla.conf` takes 2 options:
+ - site (the site like `https://bugs.xxx.net` that includes both the protocol and the domain)
+ - api_key (a Bugzilla API key)
+
+
+## Usage
+
+Run this bot as described
+[here](https://zulipchat.com/api/running-bots#running-a-bot).
+
+Use this bot with the following command
+
+`@mentioned-bot <action>` in a topic that is named `Bug 123` where 123 is the bug number
+
+### comment
+
+With no argument, by default, a new comment is added to the bug that is associated to the topic.
+For example, on topic Bug 123,
+
+you:
+
+  > @**Bugzilla** A new comment
+
+Then `A new comment` is added to bug 123
```

Comment:
Would it be worth having explicit syntax for commenting on a bug when the topic has some other format?

Reply:
For sure, it could be a nice feature, but since it was mainly aimed to work with https://github.com/lemoineat/Zulipzilla I had no use of it. And I did not want to make the bot more complex if the feature is not really required. Using a topic to track a bug record is pretty convenient.

But if anybody wants to improve it  with that feature it would be nice ! (I don't have much more time to add new features on it for the moment).